packages/frontend/src/components/common/Filters/FilterInputs/UnitOfTimeAutoComplete.tsx

```diff
@@ -114,81 +75,30 @@ const UnitOfTimeAutoComplete: FC<Props> = ({
-    onClosed,
     popoverProps,
-    disabled,
+    ...rest
 }) => (
-    <>
-        <AutocompleteMaxHeight />
-        <FieldSuggest
-            className={disabled ? 'disabled-filter' : ''}
-            disabled={disabled}
-            items={UnitOfTimeOptions({
-                isTimestamp,
-                showCompletedOptions,
-                showOptionsInPlural,
-            })}
-            itemsEqual={(value, other) =>
-                value.unitOfTime === other.unitOfTime &&
-                value.completed === other.completed
-            }
-            popoverProps={{
-                fill: true,
-                minimal: true,
-                onClosed,
-                popoverClassName: 'autocomplete-max-height',
-                ...popoverProps,
-            }}
-            itemRenderer={renderItem}
-            activeItem={
-                unitOfTime
-                    ? {
-                          label: getUnitOfTimeLabel(
-                              unitOfTime,
-                              showOptionsInPlural,
-                              completed,
-                          ),
-                          unitOfTime,
-                          completed,
-                      }
-                    : null
-            }
-            noResults={<MenuItem2 disabled text="No results." />}
-            onItemSelect={onChange}
-            itemPredicate={(
-                query: string,
-                field: UnitOfTimeOption,
-                index?: undefined | number,
-                exactMatch?: undefined | false | true,
-            ) => {
-                if (exactMatch) {
-                    return query.toLowerCase() === field.label.toLowerCase();
-                }
-                return field.label.toLowerCase().includes(query.toLowerCase());
-            }}
-        >
-            <Button
-                className={disabled ? 'disabled-filter' : ''}
-                disabled={disabled}
-                rightIcon="caret-down"
-                text={
-                    unitOfTime
-                        ? getUnitOfTimeLabel(
-                              unitOfTime,
-                              showOptionsInPlural,
-                              completed,
-                          )
-                        : 'Select value'
-                }
-                fill
-                style={{
-                    display: 'inline-flex',
-                    justifyContent: 'space-between',
-                    whiteSpace: 'nowrap',
-                }}
-            />
-        </FieldSuggest>
-    </>
+    <Select
+        searchable
+        placeholder="Select value"
+        size="xs"
+        {...rest}
+        value={completed ? `${unitOfTime}-completed` : unitOfTime}
+        data={getUnitOfTimeOptions({
+            isTimestamp,
+            showCompletedOptions,
+            showOptionsInPlural,
+        })}
+        onChange={(value) => {
+            if (value === null) return;
```

Comment:
Does this make it not clearable? Do we not want it to be eventually?

Reply:
yup, original input is not clearable + you need to pass `clearable` to a mantine component to make it clearable